File: elias/Problema0286/problema0286.py
vetor1 = []
vetor2 = []
quantidadeElementosComuns = 0
for i in range (10):
    n = int(input("Digite um número para a posição %d do primeiro vetor:  " %i))
    vetor1.append(n)
for i in range (10):
    n = int(input("Digite um número para a posição %d do segundo vetor:  " %i))
    vetor2.append(n)
print("Elementos comuns aos conjuntos:")
for i in range (10):
    for j in range (10):
        if vetor1[i] == vetor2[j] :
            print(vetor1[i])
            quantidadeElementosComuns+1
if quantidadeElementosComuns == 0 :
    print("Não há nenhum elemento comum aos conjuntos!")

File: elias/Problema0286/problema0286.java
import java.util.Scanner;

public class problema0286 {
    public static void main(String[] args) {
        Scanner input = new Scanner (System.in);
        int[] vetor1 = new int[10];
        int[] vetor2 = new int[10];
        int quantidadeElementosComuns = 0;
        for(int i=0;i<10;i++){
            System.out.print("Digite um número inteiro para a posição "+i+" do primeiro vetor: ");
            vetor1[i] = Integer.valueOf(input.nextLine());
        }
        for(int i=0;i<10;i++){
            System.out.print("Digite um número inteiro para a posição "+i+" do segundo vetor: ");
            vetor2[i] = Integer.valueOf(input.nextLine());
        }
        System.out.println("Elementos comuns aos conjuntos:");
        for(int i=0;i<10;i++){
            for(int j=0;j<10;j++){
                if(vetor1[i] == vetor2[j]){
                    System.out.println(vetor1[i]);
                    quantidadeElementosComuns++;
                }
            }
        }
        if(quantidadeElementosComuns == 0){
            System.out.println("Não há nenhum elemento comum aos conjuntos!");
        }
    }
}